### PR TITLE
Cleaning up and addressing issues with FIAM docs

### PR DESCRIPTION
**Pull request by JasonAHeron**

### CODE REVIEW

File: firebase-inappmessaging/src/main/java/com/google/firebase/inappmessaging/model/CardMessage.java [hunk 1]
```diff
@@ -148,8 +148,8 @@ public Action getSecondaryAction() {
   }
 
   /**
-   * @deprecated Use getPrimaryAction() instead. Gets the primary {@link Action} associated with
-   *     this message
+   * @deprecated This method is deprecated. Use {@link #getPrimaryAction()} or {@link
```
Comment (rachelsaunders) [inline firebase-inappmessaging/src/main/java/com/google/firebase/inappmessaging/model/CardMessage.java:151]: I _think_ that by having the @deprecated that the "This method is deprecated." bit will automatically get generated when we generate the docs.

**Comment by markarndt** [comment]
@JasonAHeron 
Please also check https://github.com/firebase/firebase-android-sdk/blob/master/firebase-inappmessaging/src/main/java/com/google/firebase/inappmessaging/FirebaseInAppMessaging.java. In master and this branch, still have reference to Instance ID.

**Pull request merged**